custom-actions/timeline/Extreme_Reach_IO.py

```diff
@@ -0,0 +1,118 @@
+#Script to set in and out points in a timeline based on the Extreme Reach delivery specs.
+def get_media_panel_custom_ui_actions():
+
+    def scope_clip(selection):
+        import flame
+        for item in selection:
+            if isinstance(item, (flame.PyClip)):
```

Comment:
No need for () pair around flame.PyLibrary, isinstance() work on either a single class name or a tuple of class name. In this case, since there is only one to check against, having () is not useful.